data_steward/resources.py

```diff
@@ -1,12 +1,24 @@
+import hashlib
 import inspect
 import os
 import csv
 import cachetools
 import json
-import vocabulary
 
 base_path = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
 
+CONCEPT = 'concept'
```

Comment:
This segment could benefit from moving these items into a constants file.  I've started a constants package structure on my id-validation-report branch.  We should do something like this, especially to clean up the possibility of circular imports that I saw in other code reviews.